test/regressions/index.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
```

Comment:
I had to start using React 18 here, otherwise the column menu wouldn't open in `ColumnMenuGridReferencesNoSnap` demo 🤷🏻‍♂️

Reply:
It also fixed this screenshot:

<img width="1460" alt="Screenshot 2023-03-02 at 15 53 57" src="https://user-images.githubusercontent.com/13808724/222464068-32c46d1c-0c3a-442c-8e21-44dc3b2492e8.png">